packages/shelter-ui/src/slider.tsx

```diff
@@ -0,0 +1,53 @@
+import { Component, JSX } from "solid-js";
+import { css, classes } from "./Slider.tsx.scss";
```

Comment:
This needs to be lowercase to work on not-Windows ;)

```suggestion
import { css, classes } from "./slider.tsx.scss";
```